kubectl-minio/cmd/resources/tenant.go

```diff
@@ -41,30 +41,26 @@ type TenantOptions struct {
 	StorageClass    string
 	KmsSecret       string
 	ConsoleSecret   string
-	CertSecret      string
 	DisableTLS      bool
 	ImagePullSecret string
 }
 
 // Validate Tenant Options
 func (t TenantOptions) Validate() error {
-	if t.Name == "" {
-		return errors.New("--name flag is required for tenant creation")
-	}
 	if t.Servers == 0 {
-		return errors.New("--servers flag is required for tenant creation")
+		return errors.New("--servers flag is required")
```

Comment:
Actually we might unify this as

if t.Servers <=0 { return errors.new("--servers must be greater than or equal to 1") }
or "--servers minimum value is 1"

Reply:
Same pattern for the other flags validations if this makes sense.